packages/buidler-core/package-lock.json

```diff
@@ -1584,14 +1584,6 @@
 			"resolved": "https://registry.npmjs.org/invert-kv/-/invert-kv-1.0.0.tgz",
 			"integrity": "sha1-EEqOSqym09jNFXqO+L+rLXo//bY="
 		},
```

Comment:
Why is `io-ts: 1.10.4` not here?

Reply:
No idea, but this file will be deleted by #322